lambdas/service/app.py

```diff
@@ -821,36 +825,84 @@ def get_order():
     return {'order': Plugin.load().service_config().order_config}
 
 
-@app.route('/manifest/files', methods=['GET'], cors=True)
-def start_manifest_generation():
-    """
-    Initiate and check status of a manifest generation job, returning a either a 301 or 302 response
-    redirecting to either the location of the manifest or a URL to re-check the status of the manifest job.
+manifest_filters = {
```

Comment:
We need to make sure that the meta domain variables such as this one are name in such a way that their "meta-ness" is obvious to avoid confusion and naming conflicts. The suffix `_spec` already been established for this purpose.

```suggestion
manifest_filter_spec = {
```

Reply:
Secondly, the filter argument is universal so this can be generalized to

```suggestion
filter_spec = {
```